Sources/EpoxyCore/SwiftUI/LayoutUtilities/SwiftUIMeasurementContainer.swift

```diff
@@ -58,14 +64,18 @@ public final class SwiftUIMeasurementContainer<SwiftUIView, UIViewType>: UIView
     // We need to re-measure the view whenever the size of the bounds change, as the previous size
     // will be incorrect.
     if bounds.size != latestMeasurementBoundsSize {
-      measureView()
+      if measureView().changed {
```

Comment:
if our sizing behavior is `.intrinsicSize` we can avoid doing this additional measurement.

Reply:
Good call!